Bottom: 503b8d85341e58292dcaa7af38191083b414ee5d
Top:    69c655994e71af522643f99474add76742e8cc73
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-02 14:13:30 +0000

hid/common/hidgl.c: Make (polygon->clipped == NULL) test silent

This is the case for other implementations of this function, so we
should aim to be consistent.


---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index d9fb0ad..73e85ce 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -626,10 +626,7 @@ hidgl_fill_pcb_polygon (PolygonType *poly, const BoxType *clip_box, double scale
   global_scale = scale;
 
   if (poly->Clipped == NULL)
-    {
-      fprintf (stderr, "hidgl_fill_pcb_polygon: poly->Clipped == NULL\n");
-      return;
-    }
+    return;
 
   stencil_bit = hidgl_assign_clear_stencil_bit ();
   if (!stencil_bit)
